verify-service-provider.yml

```diff
@@ -29,6 +29,8 @@ serviceEntityIds: ${SERVICE_ENTITY_IDS:-[]}
 # To use the Verify Service Provider with multiple services, add their entity IDs here, e.g.
 # '["service-one-entity-id", "service-two-entity-id"]'
 
+# This applies if you're connecting using only the VSP
```

Comment:
Also, it's optional if you only have one service (i.e. only one entry in serviceEntityIds above).  It's only multi-tenanted RPs who need to configure a hashingEntityId.